From: =?utf-8?q?Fabian_Gr=C3=BCnbichler?= <f.gruenbichler@proxmox.com>
Date: Wed, 6 Sep 2023 13:23:24 -0600
Subject: d-0021-vendor-remove-windows-dependencies
MIME-Version: 1.0
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

use something like

 find vendor -iname Cargo.toml -exec grep -H -n -e 'schannel' -e 'windows-sys' -e 'winapi' -e 'ntapi' -e 'wincon' -e 'winreg' -e 'windows' -e 'winsplit' {} \;

to find dependencies on windows targets in vendored crates. you will likely
need to remove some hunks from this patch after pruning dependencies, since
hopefully a few of the crates patched during early rebasing are eliminated.

windows-bindgen and windows-metadata should not be removed, they are needed for
the build and don't pull in windows-sys and friends.

Forwarded: not-needed

Signed-off-by: Fabian Grünbichler <f.gruenbichler@proxmox.com>
---
 vendor/anstream-0.6.21/Cargo.toml             |  6 --
 vendor/anstream-0.6.21/Cargo.toml.orig        |  3 +-
 vendor/anstyle-query-1.1.5/Cargo.toml         |  7 ---
 vendor/backtrace-0.3.75/Cargo.toml            |  3 -
 vendor/chrono-0.4.42/Cargo.toml               |  9 ---
 vendor/colored-2.2.0/Cargo.toml               |  7 ---
 vendor/colored-3.0.0/Cargo.toml               |  7 ---
 vendor/console-0.15.11/Cargo.toml             | 12 ----
 vendor/ctrlc-3.5.1/Cargo.toml                 | 18 ------
 vendor/curl-0.4.49/Cargo.toml                 | 15 -----
 vendor/curl-sys-0.4.84+curl-8.17.0/Cargo.toml |  6 --
 vendor/dbus-0.9.9/Cargo.toml                  |  4 --
 vendor/dirs-sys-0.5.0/Cargo.toml              |  9 ---
 vendor/dirs-sys-next-0.1.2/Cargo.toml         |  3 -
 vendor/dlmalloc-0.2.11/Cargo.toml             |  9 ---
 vendor/env_logger-0.11.8/Cargo.toml           |  2 +-
 vendor/errno-0.3.14/Cargo.toml                |  7 ---
 vendor/filetime-0.2.26/Cargo.toml             |  7 ---
 vendor/gix-discover-0.42.0/Cargo.toml         |  3 -
 vendor/gix-path-0.10.22/Cargo.toml            | 10 ---
 vendor/gix-sec-0.12.2/Cargo.toml              | 13 ----
 vendor/home-0.5.12/Cargo.toml                 |  8 ---
 vendor/hyper-util-0.1.18/Cargo.toml           | 10 ---
 vendor/iana-time-zone-0.1.64/Cargo.toml       | 12 ----
 vendor/ignore-0.4.25/Cargo.toml               |  3 -
 vendor/is_executable-1.0.5/Cargo.toml         |  3 -
 vendor/jiff-0.2.16/Cargo.toml                 | 10 ---
 vendor/libloading-0.8.9/Cargo.toml            |  7 ---
 vendor/libssh2-sys-0.3.1/Cargo.toml           |  8 ---
 vendor/mio-1.1.0/Cargo.toml                   | 16 -----
 vendor/notify-8.2.0/Cargo.toml                | 14 -----
 vendor/nu-ansi-term-0.50.3/Cargo.toml         | 10 ---
 vendor/onig-6.5.1/Cargo.toml                  |  3 -
 vendor/opener-0.7.2/Cargo.toml                | 12 ----
 vendor/opener-0.8.3/Cargo.toml                | 12 ----
 vendor/os_info-3.13.0/Cargo.toml              | 34 -----------
 vendor/parking_lot_core-0.9.12/Cargo.toml     |  2 -
 vendor/portable-atomic-1.11.1/Cargo.toml      |  7 ---
 vendor/process-wrap-8.2.1/Cargo.toml          | 12 ----
 vendor/rusqlite-0.37.0/Cargo.toml             | 18 ------
 vendor/rustix-1.1.2/Cargo.toml                | 15 -----
 vendor/same-file-1.0.6/Cargo.toml             |  2 -
 vendor/snapbox-0.6.23/Cargo.toml              |  6 --
 vendor/socket2-0.6.1/Cargo.toml               |  9 ---
 vendor/stacker-0.1.21/Cargo.toml              |  7 ---
 vendor/sysinfo-0.37.2/Cargo.toml              | 87 +--------------------------
 vendor/tempfile-3.23.0/Cargo.toml             |  7 ---
 vendor/term-0.7.0/Cargo.toml                  |  6 --
 vendor/termcolor-1.4.1/Cargo.toml             |  3 -
 vendor/terminal_size-0.4.3/Cargo.toml         |  7 ---
 vendor/termize-0.2.1/Cargo.toml               |  7 ---
 vendor/tokio-1.48.0/Cargo.toml                | 21 -------
 vendor/walkdir-2.5.0/Cargo.toml               |  3 -
 vendor/wasm-component-ld-0.5.19/Cargo.toml    |  7 ---
 vendor/yansi-term-0.1.2/Cargo.toml            |  3 -
 55 files changed, 3 insertions(+), 548 deletions(-)

diff --git a/vendor/anstream-0.6.21/Cargo.toml b/vendor/anstream-0.6.21/Cargo.toml
index 7c4249d..82d233f 100644
--- a/vendor/anstream-0.6.21/Cargo.toml
+++ b/vendor/anstream-0.6.21/Cargo.toml
@@ -89,10 +89,8 @@ exactly = 1
 auto = ["dep:anstyle-query"]
 default = [
     "auto",
-    "wincon",
 ]
 test = []
-wincon = ["dep:anstyle-wincon"]
 
 [lib]
 name = "anstream"
@@ -142,10 +140,6 @@ version = "1.7.0"
 [dev-dependencies.strip-ansi-escapes]
 version = "0.2.1"
 
-[target."cfg(windows)".dependencies.anstyle-wincon]
-version = "3.0.5"
-optional = true
-
 [lints.clippy]
 bool_assert_comparison = "allow"
 branches_sharing_code = "allow"
diff --git a/vendor/anstream-0.6.21/Cargo.toml.orig b/vendor/anstream-0.6.21/Cargo.toml.orig
index 942ac10..80b097c 100644
--- a/vendor/anstream-0.6.21/Cargo.toml.orig
+++ b/vendor/anstream-0.6.21/Cargo.toml.orig
@@ -24,9 +24,8 @@ pre-release-replacements = [
 ]
 
 [features]
-default = ["auto", "wincon"]
+default = ["auto"]
 auto = ["dep:anstyle-query"]
-wincon = ["dep:anstyle-wincon"]
 # Enable in `dev-dependencies` to make sure output is captured for tests
 test = []
 
diff --git a/vendor/anstyle-query-1.1.5/Cargo.toml b/vendor/anstyle-query-1.1.5/Cargo.toml
index 1268520..012d9be 100644
--- a/vendor/anstyle-query-1.1.5/Cargo.toml
+++ b/vendor/anstyle-query-1.1.5/Cargo.toml
@@ -93,13 +93,6 @@ path = "src/lib.rs"
 name = "query"
 path = "examples/query.rs"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.60.2, <0.62"
-features = [
-    "Win32_System_Console",
-    "Win32_Foundation",
-]
-
 [lints.clippy]
 bool_assert_comparison = "allow"
 branches_sharing_code = "allow"
diff --git a/vendor/backtrace-0.3.75/Cargo.toml b/vendor/backtrace-0.3.75/Cargo.toml
index 541be60..f6edc47 100644
--- a/vendor/backtrace-0.3.75/Cargo.toml
+++ b/vendor/backtrace-0.3.75/Cargo.toml
@@ -120,9 +120,6 @@ optional = true
 [dev-dependencies.libloading]
 version = "0.8"
 
-[target.'cfg(any(windows, target_os = "cygwin"))'.dependencies.windows-targets]
-version = "0.52.6"
-
 [target.'cfg(not(all(windows, target_env = "msvc", not(target_vendor = "uwp"))))'.dependencies.addr2line]
 version = "0.24.0"
 default-features = false
diff --git a/vendor/chrono-0.4.42/Cargo.toml b/vendor/chrono-0.4.42/Cargo.toml
index cc69ee3..9ecc19d 100644
--- a/vendor/chrono-0.4.42/Cargo.toml
+++ b/vendor/chrono-0.4.42/Cargo.toml
@@ -58,7 +58,6 @@ features = ["serde"]
 __internal_bench = []
 alloc = []
 clock = [
-    "winapi",
     "iana-time-zone",
     "now",
 ]
@@ -95,7 +94,6 @@ wasmbind = [
     "wasm-bindgen",
     "js-sys",
 ]
-winapi = ["windows-link"]
 
 [lib]
 name = "chrono"
@@ -164,10 +162,3 @@ version = "0.3"
 version = "0.1.45"
 features = ["fallback"]
 optional = true
-
-[target."cfg(windows)".dependencies.windows-link]
-version = "0.2"
-optional = true
-
-[target."cfg(windows)".dev-dependencies.windows-bindgen]
-version = "0.63"
diff --git a/vendor/colored-2.2.0/Cargo.toml b/vendor/colored-2.2.0/Cargo.toml
index 500eb97..2e9ce71 100644
--- a/vendor/colored-2.2.0/Cargo.toml
+++ b/vendor/colored-2.2.0/Cargo.toml
@@ -80,10 +80,3 @@ version = "1"
 
 [features]
 no-color = []
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.48,<=0.59"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
diff --git a/vendor/colored-3.0.0/Cargo.toml b/vendor/colored-3.0.0/Cargo.toml
index 4f7b53a..f64fdcb 100644
--- a/vendor/colored-3.0.0/Cargo.toml
+++ b/vendor/colored-3.0.0/Cargo.toml
@@ -77,10 +77,3 @@ version = "1"
 
 [features]
 no-color = []
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.48,<=0.59"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
diff --git a/vendor/console-0.15.11/Cargo.toml b/vendor/console-0.15.11/Cargo.toml
index 785d469..31faf42 100644
--- a/vendor/console-0.15.11/Cargo.toml
+++ b/vendor/console-0.15.11/Cargo.toml
@@ -92,15 +92,3 @@ default-features = false
 
 [dev-dependencies.regex]
 version = "1.4.2"
-
-[target."cfg(windows)".dependencies.encode_unicode]
-version = "1"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.59"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_UI_Input_KeyboardAndMouse",
-]
diff --git a/vendor/ctrlc-3.5.1/Cargo.toml b/vendor/ctrlc-3.5.1/Cargo.toml
index dd42131..4bb0f09 100644
--- a/vendor/ctrlc-3.5.1/Cargo.toml
+++ b/vendor/ctrlc-3.5.1/Cargo.toml
@@ -76,21 +76,3 @@ version = "0.3"
 version = "0.30"
 features = ["signal"]
 default-features = false
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Threading",
-    "Win32_Security",
-    "Win32_System_Console",
-]
-
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Storage_FileSystem",
-    "Win32_Foundation",
-    "Win32_System_IO",
-    "Win32_System_Console",
-]
diff --git a/vendor/curl-0.4.49/Cargo.toml b/vendor/curl-0.4.49/Cargo.toml
index e491ab9..124295b 100644
--- a/vendor/curl-0.4.49/Cargo.toml
+++ b/vendor/curl-0.4.49/Cargo.toml
@@ -49,10 +49,6 @@ ssl = [
 static-curl = ["curl-sys/static-curl"]
 static-ssl = ["curl-sys/static-ssl"]
 upkeep_7_62_0 = ["curl-sys/upkeep_7_62_0"]
-windows-static-ssl = [
-    "static-curl",
-    "curl-sys/windows-static-ssl",
-]
 zlib-ng-compat = [
     "curl-sys/zlib-ng-compat",
     "static-curl",
@@ -145,14 +141,3 @@ optional = true
 [target."cfg(all(unix))".dependencies.openssl-sys]
 version = "0.9.64"
 optional = true
-
-[target.'cfg(target_env = "msvc")'.dependencies.schannel]
-version = "0.1.13"
-
-[target.'cfg(target_env = "msvc")'.dependencies.windows-sys]
-version = "0.59"
-features = [
-    "Win32_Foundation",
-    "Win32_System_LibraryLoader",
-    "Win32_Security_Cryptography",
-]
diff --git a/vendor/curl-sys-0.4.84+curl-8.17.0/Cargo.toml b/vendor/curl-sys-0.4.84+curl-8.17.0/Cargo.toml
index 3ed5f51..1fb09d2 100644
--- a/vendor/curl-sys-0.4.84+curl-8.17.0/Cargo.toml
+++ b/vendor/curl-sys-0.4.84+curl-8.17.0/Cargo.toml
@@ -83,9 +83,3 @@ version = "0.3.3"
 version = "0.9.64"
 optional = true
 
-[target.'cfg(target_env = "msvc")'.build-dependencies.vcpkg]
-version = "0.2"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.59"
-features = ["Win32_Networking_WinSock"]
diff --git a/vendor/dbus-0.9.9/Cargo.toml b/vendor/dbus-0.9.9/Cargo.toml
index 396557d..c8a2c0c 100644
--- a/vendor/dbus-0.9.9/Cargo.toml
+++ b/vendor/dbus-0.9.9/Cargo.toml
@@ -103,7 +103,3 @@ version = "0.2.6"
 
 [dev-dependencies.tempfile]
 version = "3"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.59.0"
-features = ["Win32_Networking_WinSock"]
diff --git a/vendor/dirs-sys-0.5.0/Cargo.toml b/vendor/dirs-sys-0.5.0/Cargo.toml
index 7b64137..ffea536 100644
--- a/vendor/dirs-sys-0.5.0/Cargo.toml
+++ b/vendor/dirs-sys-0.5.0/Cargo.toml
@@ -37,12 +37,3 @@ default-features = false
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">= 0.59.0"
-features = [
-    "Win32_UI_Shell",
-    "Win32_Foundation",
-    "Win32_Globalization",
-    "Win32_System_Com",
-]
diff --git a/vendor/dirs-sys-next-0.1.2/Cargo.toml b/vendor/dirs-sys-next-0.1.2/Cargo.toml
index e9d8d0c..acb2eb7 100644
--- a/vendor/dirs-sys-next-0.1.2/Cargo.toml
+++ b/vendor/dirs-sys-next-0.1.2/Cargo.toml
@@ -25,8 +25,5 @@ version = "0.4.0"
 default-features = false
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["knownfolders", "objbase", "shlobj", "winbase", "winerror"]
 [badges.maintenance]
 status = "as-is"
diff --git a/vendor/dlmalloc-0.2.11/Cargo.toml b/vendor/dlmalloc-0.2.11/Cargo.toml
index b9babb4..849de6f 100644
--- a/vendor/dlmalloc-0.2.11/Cargo.toml
+++ b/vendor/dlmalloc-0.2.11/Cargo.toml
@@ -73,14 +73,5 @@ features = ["small_rng"]
 version = "0.2"
 default-features = false
 
-[target.'cfg(target_os = "windows")'.dependencies.windows-sys]
-version = ">=0.52.0, <=0.60.*"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Memory",
-    "Win32_System_Threading",
-    "Win32_System_SystemInformation",
-]
-
 [profile.release]
 debug-assertions = true
diff --git a/vendor/env_logger-0.11.8/Cargo.toml b/vendor/env_logger-0.11.8/Cargo.toml
index 48f6224..b478092 100644
--- a/vendor/env_logger-0.11.8/Cargo.toml
+++ b/vendor/env_logger-0.11.8/Cargo.toml
@@ -161,7 +161,7 @@ harness = false
 
 [dependencies.anstream]
 version = "0.6.11"
-features = ["wincon"]
+features = []
 optional = true
 default-features = false
 
diff --git a/vendor/errno-0.3.14/Cargo.toml b/vendor/errno-0.3.14/Cargo.toml
index ff3dfa9..81a2125 100644
--- a/vendor/errno-0.3.14/Cargo.toml
+++ b/vendor/errno-0.3.14/Cargo.toml
@@ -54,13 +54,6 @@ default-features = false
 version = "0.2"
 default-features = false
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.52, <0.62"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Diagnostics_Debug",
-]
-
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
diff --git a/vendor/filetime-0.2.26/Cargo.toml b/vendor/filetime-0.2.26/Cargo.toml
index d6ebc24..3731c50 100644
--- a/vendor/filetime-0.2.26/Cargo.toml
+++ b/vendor/filetime-0.2.26/Cargo.toml
@@ -48,10 +48,3 @@ version = "0.1.0"
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.27"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.60.0"
-features = [
-    "Win32_Foundation",
-    "Win32_Storage_FileSystem",
-]
diff --git a/vendor/gix-discover-0.42.0/Cargo.toml b/vendor/gix-discover-0.42.0/Cargo.toml
index 3e0faa9..a30f525 100644
--- a/vendor/gix-discover-0.42.0/Cargo.toml
+++ b/vendor/gix-discover-0.42.0/Cargo.toml
@@ -74,9 +74,6 @@ version = "3.23.0"
 [target.'cfg(target_os = "macos")'.dev-dependencies.defer]
 version = "0.2.1"
 
-[target."cfg(windows)".dependencies.dunce]
-version = "1.0.3"
-
 [lints.clippy]
 bool_to_int_with_if = "allow"
 borrow_as_ptr = "allow"
diff --git a/vendor/gix-path-0.10.22/Cargo.toml b/vendor/gix-path-0.10.22/Cargo.toml
index 7f6ce20..87fb33f 100644
--- a/vendor/gix-path-0.10.22/Cargo.toml
+++ b/vendor/gix-path-0.10.22/Cargo.toml
@@ -53,16 +53,6 @@ version = "2.0.17"
 version = "3.1.0"
 default-features = false
 
-[target."cfg(windows)".dev-dependencies.windows]
-version = "0.62.1"
-features = [
-    "Win32_System_Com",
-    "Win32_System_Threading",
-    "Win32_UI_Shell",
-]
-
-[target."cfg(windows)".dev-dependencies.winreg]
-version = "0.55.0"
 
 [lints.clippy]
 bool_to_int_with_if = "allow"
diff --git a/vendor/gix-sec-0.12.2/Cargo.toml b/vendor/gix-sec-0.12.2/Cargo.toml
index 119bc1b..074c2a7 100644
--- a/vendor/gix-sec-0.12.2/Cargo.toml
+++ b/vendor/gix-sec-0.12.2/Cargo.toml
@@ -67,19 +67,6 @@ version = "3.23.0"
 [target."cfg(not(windows))".dependencies.libc]
 version = "0.2.176"
 
-[target."cfg(windows)".dependencies.gix-path]
-version = "^0.10.21"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.1"
-features = [
-    "Win32_Foundation",
-    "Win32_Security_Authorization",
-    "Win32_Storage_FileSystem",
-    "Win32_System_Memory",
-    "Win32_System_Threading",
-]
-
 [lints.clippy]
 bool_to_int_with_if = "allow"
 borrow_as_ptr = "allow"
diff --git a/vendor/home-0.5.12/Cargo.toml b/vendor/home-0.5.12/Cargo.toml
index 3361c27..7f15d06 100644
--- a/vendor/home-0.5.12/Cargo.toml
+++ b/vendor/home-0.5.12/Cargo.toml
@@ -40,14 +40,6 @@ resolver = "2"
 name = "home"
 path = "src/lib.rs"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_UI_Shell",
-    "Win32_System_Com",
-]
-
 [lints.clippy]
 dbg_macro = "warn"
 disallowed_methods = "warn"
diff --git a/vendor/hyper-util-0.1.18/Cargo.toml b/vendor/hyper-util-0.1.18/Cargo.toml
index 8babba4..878a3e5 100644
--- a/vendor/hyper-util-0.1.18/Cargo.toml
+++ b/vendor/hyper-util-0.1.18/Cargo.toml
@@ -68,8 +68,6 @@ client-proxy = [
     "dep:percent-encoding",
 ]
 client-proxy-system = [
-    "dep:system-configuration",
-    "dep:windows-registry",
 ]
 default = []
 full = [
@@ -239,11 +237,3 @@ version = "0.4"
 
 [target.'cfg(any(target_os = "linux", target_os = "macos"))'.dev-dependencies.pnet_datalink]
 version = "0.35.0"
-
-[target.'cfg(target_os = "macos")'.dependencies.system-configuration]
-version = ">=0.5, <0.7"
-optional = true
-
-[target."cfg(windows)".dependencies.windows-registry]
-version = ">=0.3, <0.7"
-optional = true
diff --git a/vendor/iana-time-zone-0.1.64/Cargo.toml b/vendor/iana-time-zone-0.1.64/Cargo.toml
index 2020d62..3df4bcd 100644
--- a/vendor/iana-time-zone-0.1.64/Cargo.toml
+++ b/vendor/iana-time-zone-0.1.64/Cargo.toml
@@ -79,15 +79,3 @@ features = ["js"]
 
 [target.'cfg(all(target_arch = "wasm32", target_os = "unknown"))'.dev-dependencies.wasm-bindgen-test]
 version = "0.3.46"
-
-[target.'cfg(target_os = "android")'.dependencies.android_system_properties]
-version = "0.1.5"
-
-[target.'cfg(target_os = "haiku")'.dependencies.iana-time-zone-haiku]
-version = "0.1.1"
-
-[target.'cfg(target_os = "windows")'.dependencies.windows-core]
-version = ">=0.56, <=0.62"
-
-[target.'cfg(target_vendor = "apple")'.dependencies.core-foundation-sys]
-version = "0.8.6"
diff --git a/vendor/ignore-0.4.25/Cargo.toml b/vendor/ignore-0.4.25/Cargo.toml
index aaa209a..d117779 100644
--- a/vendor/ignore-0.4.25/Cargo.toml
+++ b/vendor/ignore-0.4.25/Cargo.toml
@@ -95,6 +95,3 @@ default-features = false
 
 [dev-dependencies.crossbeam-channel]
 version = "0.5.15"
-
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.2"
diff --git a/vendor/is_executable-1.0.5/Cargo.toml b/vendor/is_executable-1.0.5/Cargo.toml
index 098c6a8..dd2509a 100644
--- a/vendor/is_executable-1.0.5/Cargo.toml
+++ b/vendor/is_executable-1.0.5/Cargo.toml
@@ -44,6 +44,3 @@ path = "tests/tests.rs"
 [dev-dependencies.diff]
 version = "0.1.10"
 
-[target.'cfg(target_os = "windows")'.dependencies.windows-sys]
-version = "0.60"
-features = ["Win32_Storage_FileSystem"]
diff --git a/vendor/jiff-0.2.16/Cargo.toml b/vendor/jiff-0.2.16/Cargo.toml
index 2a225ba..24aaf8c 100644
--- a/vendor/jiff-0.2.16/Cargo.toml
+++ b/vendor/jiff-0.2.16/Cargo.toml
@@ -90,7 +90,6 @@ std = [
 tz-fat = ["jiff-static?/tz-fat"]
 tz-system = [
     "std",
-    "dep:windows-sys",
 ]
 tzdb-concatenated = ["std"]
 tzdb-zoneinfo = ["std"]
@@ -193,15 +192,6 @@ default-features = false
 version = "0.2.4"
 default-features = false
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.52.0, <=0.61.*"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Time",
-]
-optional = true
-default-features = false
-
 [lints.rust.unexpected_cfgs]
 level = "allow"
 priority = 0
diff --git a/vendor/libloading-0.8.9/Cargo.toml b/vendor/libloading-0.8.9/Cargo.toml
index 2c8eb67..ff74d06 100644
--- a/vendor/libloading-0.8.9/Cargo.toml
+++ b/vendor/libloading-0.8.9/Cargo.toml
@@ -74,13 +74,6 @@ version = "1.1"
 [target."cfg(unix)".dependencies.cfg-if]
 version = "1"
 
-[target."cfg(windows)".dependencies.windows-link]
-version = "0.2"
-
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.61"
-features = ["Win32_Foundation"]
-
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
diff --git a/vendor/libssh2-sys-0.3.1/Cargo.toml b/vendor/libssh2-sys-0.3.1/Cargo.toml
index 55ee005..0d06451 100644
--- a/vendor/libssh2-sys-0.3.1/Cargo.toml
+++ b/vendor/libssh2-sys-0.3.1/Cargo.toml
@@ -49,16 +49,8 @@ version = "1.0.25"
 version = "0.3.11"
 
 [features]
-openssl-on-win32 = ["openssl-sys"]
 vendored-openssl = ["openssl-sys/vendored"]
 zlib-ng-compat = ["libz-sys/zlib-ng"]
 
-[target.'cfg(target_env = "msvc")'.build-dependencies.vcpkg]
-version = "0.2"
-
 [target."cfg(unix)".dependencies.openssl-sys]
 version = "0.9.35"
-
-[target."cfg(windows)".dependencies.openssl-sys]
-version = "0.9.35"
-optional = true
diff --git a/vendor/mio-1.1.0/Cargo.toml b/vendor/mio-1.1.0/Cargo.toml
index 1ff8995..7bdff88 100644
--- a/vendor/mio-1.1.0/Cargo.toml
+++ b/vendor/mio-1.1.0/Cargo.toml
@@ -80,8 +80,6 @@ default = ["log"]
 net = []
 os-ext = [
     "os-poll",
-    "windows-sys/Win32_System_Pipes",
-    "windows-sys/Win32_Security",
 ]
 os-poll = []
 
@@ -136,20 +134,6 @@ version = "0.11.0"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.159"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Wdk_Foundation",
-    "Wdk_Storage_FileSystem",
-    "Wdk_System_IO",
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-    "Win32_Storage_FileSystem",
-    "Win32_Security",
-    "Win32_System_IO",
-    "Win32_System_WindowsProgramming",
-]
-
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
diff --git a/vendor/notify-8.2.0/Cargo.toml b/vendor/notify-8.2.0/Cargo.toml
index ba52fc3..61624a7 100644
--- a/vendor/notify-8.2.0/Cargo.toml
+++ b/vendor/notify-8.2.0/Cargo.toml
@@ -90,9 +90,6 @@ default-features = false
 version = "1.0"
 features = ["os-ext"]
 
-[target."cfg(target_os = \"windows\")".dev-dependencies.trash]
-version = "5.2.2"
-
 [target."cfg(target_os=\"macos\")".dependencies.bitflags]
 version = "2.7.0"
 
@@ -108,14 +105,3 @@ optional = true
 version = "1.0"
 features = ["os-ext"]
 optional = true
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.60.1"
-features = [
-    "Win32_System_Threading",
-    "Win32_Foundation",
-    "Win32_Storage_FileSystem",
-    "Win32_Security",
-    "Win32_System_WindowsProgramming",
-    "Win32_System_IO",
-]
diff --git a/vendor/nu-ansi-term-0.50.3/Cargo.toml b/vendor/nu-ansi-term-0.50.3/Cargo.toml
index 27fad0c..3801b07 100644
--- a/vendor/nu-ansi-term-0.50.3/Cargo.toml
+++ b/vendor/nu-ansi-term-0.50.3/Cargo.toml
@@ -80,13 +80,3 @@ version = "0.3.3"
 
 [dev-dependencies.serde_json]
 version = "1.0.94"
-
-[target."cfg(windows)".dependencies.windows]
-version = ">=0.59, <=0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-    "Win32_Storage_FileSystem",
-    "Win32_Security",
-]
-package = "windows-sys"
diff --git a/vendor/onig-6.5.1/Cargo.toml b/vendor/onig-6.5.1/Cargo.toml
index a218b18..52ee579 100644
--- a/vendor/onig-6.5.1/Cargo.toml
+++ b/vendor/onig-6.5.1/Cargo.toml
@@ -95,6 +95,3 @@ version = "1.12"
 [dependencies.onig_sys]
 version = "69.9.1"
 default-features = false
-
-[target."cfg(windows)".dependencies.libc]
-version = "0.2"
diff --git a/vendor/opener-0.7.2/Cargo.toml b/vendor/opener-0.7.2/Cargo.toml
index 5639d3d..eff5c32 100644
--- a/vendor/opener-0.7.2/Cargo.toml
+++ b/vendor/opener-0.7.2/Cargo.toml
@@ -48,7 +48,6 @@ default = ["dbus-vendored"]
 reveal = [
     "dep:url",
     "dep:dbus",
-    "windows-sys/Win32_System_Com",
 ]
 
 [target.'cfg(target_os = "linux")'.dependencies.bstr]
@@ -62,16 +61,5 @@ optional = true
 version = "2"
 optional = true
 
-[target."cfg(windows)".dependencies.normpath]
-version = "1"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.59"
-features = [
-    "Win32_Foundation",
-    "Win32_UI_Shell",
-    "Win32_UI_WindowsAndMessaging",
-]
-
 [badges.maintenance]
 status = "passively-maintained"
diff --git a/vendor/opener-0.8.3/Cargo.toml b/vendor/opener-0.8.3/Cargo.toml
index 9e01879..d849141 100644
--- a/vendor/opener-0.8.3/Cargo.toml
+++ b/vendor/opener-0.8.3/Cargo.toml
@@ -46,8 +46,6 @@ status = "passively-maintained"
 reveal = [
     "dep:url",
     "dep:zbus",
-    "windows-sys/Win32_System_Com",
-    "windows-sys/Win32_UI_Shell_Common",
 ]
 
 [lib]
@@ -65,13 +63,3 @@ optional = true
 version = "5"
 features = ["url"]
 optional = true
-
-[target."cfg(windows)".dependencies.normpath]
-version = "1"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.60"
-features = [
-    "Win32_UI_Shell",
-    "Win32_UI_WindowsAndMessaging",
-]
diff --git a/vendor/os_info-3.13.0/Cargo.toml b/vendor/os_info-3.13.0/Cargo.toml
index d9bb138..68548cc 100644
--- a/vendor/os_info-3.13.0/Cargo.toml
+++ b/vendor/os_info-3.13.0/Cargo.toml
@@ -68,37 +68,3 @@ version = "1"
 [target.'cfg(any(target_os = "aix", target_os = "dragonfly", target_os = "freebsd", target_os = "illumos", target_os = "linux", target_os = "macos", target_os = "netbsd", target_os = "openbsd", target_os = "cygwin"))'.dependencies.nix]
 version = "0.30"
 features = ["feature"]
-
-[target.'cfg(target_os = "android")'.dependencies.android_system_properties]
-version = "0.1"
-
-[target.'cfg(target_os = "ios")'.dependencies.objc2]
-version = "0.6"
-
-[target.'cfg(target_os = "ios")'.dependencies.objc2-foundation]
-version = "0.3"
-features = ["NSString"]
-
-[target.'cfg(target_os = "ios")'.dependencies.objc2-ui-kit]
-version = "0.3"
-
-[target.'cfg(target_os = "macos")'.dependencies.objc2-foundation]
-version = "0.3"
-features = [
-    "NSData",
-    "NSError",
-    "NSEnumerator",
-    "NSString",
-]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_System_LibraryLoader",
-    "Win32_System_Registry",
-    "Win32_System_SystemInformation",
-    "Win32_System_SystemServices",
-    "Win32_System_Threading",
-    "Win32_UI_WindowsAndMessaging",
-]
diff --git a/vendor/parking_lot_core-0.9.12/Cargo.toml b/vendor/parking_lot_core-0.9.12/Cargo.toml
index ecd9178..475e0e1 100644
--- a/vendor/parking_lot_core-0.9.12/Cargo.toml
+++ b/vendor/parking_lot_core-0.9.12/Cargo.toml
@@ -68,5 +68,3 @@ version = "0.5"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.95"
 
-[target."cfg(windows)".dependencies.windows-link]
-version = "0.2.0"
diff --git a/vendor/portable-atomic-1.11.1/Cargo.toml b/vendor/portable-atomic-1.11.1/Cargo.toml
index b337812..c4f08f9 100644
--- a/vendor/portable-atomic-1.11.1/Cargo.toml
+++ b/vendor/portable-atomic-1.11.1/Cargo.toml
@@ -109,13 +109,6 @@ version = "1"
 [target."cfg(unix)".dev-dependencies.libc]
 version = "=0.2.163"
 
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.59"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Threading",
-]
-
 [lints.clippy]
 all = "warn"
 as_ptr_cast_mut = "warn"
diff --git a/vendor/process-wrap-8.2.1/Cargo.toml b/vendor/process-wrap-8.2.1/Cargo.toml
index 43c7d65..a2151d1 100644
--- a/vendor/process-wrap-8.2.1/Cargo.toml
+++ b/vendor/process-wrap-8.2.1/Cargo.toml
@@ -44,8 +44,6 @@ all-features = true
 
 [features]
 creation-flags = [
-    "dep:windows",
-    "windows/Win32_System_Threading",
 ]
 default = [
     "creation-flags",
@@ -57,12 +55,6 @@ default = [
 ]
 downcasting = []
 job-object = [
-    "dep:windows",
-    "windows/Win32_Security",
-    "windows/Win32_System_Diagnostics_ToolHelp",
-    "windows/Win32_System_IO",
-    "windows/Win32_System_JobObjects",
-    "windows/Win32_System_Threading",
 ]
 kill-on-drop = []
 process-group = []
@@ -144,7 +136,3 @@ features = [
 ]
 optional = true
 default-features = false
-
-[target."cfg(windows)".dependencies.windows]
-version = "0.61.1"
-optional = true
diff --git a/vendor/rusqlite-0.37.0/Cargo.toml b/vendor/rusqlite-0.37.0/Cargo.toml
index eef4d05..df49818 100644
--- a/vendor/rusqlite-0.37.0/Cargo.toml
+++ b/vendor/rusqlite-0.37.0/Cargo.toml
@@ -55,7 +55,6 @@ rustdoc-args = [
 
 [package.metadata.playground]
 all-features = false
-features = ["bundled-full"]
 
 [badges.appveyor]
 repository = "rusqlite/rusqlite"
@@ -74,23 +73,6 @@ array = [
 backup = []
 blob = []
 buildtime_bindgen = ["libsqlite3-sys/buildtime_bindgen"]
-bundled = [
-    "libsqlite3-sys/bundled",
-    "modern_sqlite",
-]
-bundled-full = [
-    "modern-full",
-    "bundled",
-]
-bundled-sqlcipher = [
-    "libsqlite3-sys/bundled-sqlcipher",
-    "bundled",
-]
-bundled-sqlcipher-vendored-openssl = [
-    "libsqlite3-sys/bundled-sqlcipher-vendored-openssl",
-    "bundled-sqlcipher",
-]
-bundled-windows = ["libsqlite3-sys/bundled-windows"]
 collation = []
 column_decltype = []
 column_metadata = ["libsqlite3-sys/column_metadata"]
diff --git a/vendor/rustix-1.1.2/Cargo.toml b/vendor/rustix-1.1.2/Cargo.toml
index 1e8347a..b3cc363 100644
--- a/vendor/rustix-1.1.2/Cargo.toml
+++ b/vendor/rustix-1.1.2/Cargo.toml
@@ -235,21 +235,6 @@ version = "0.3.10"
 default-features = false
 package = "errno"
 
-[target."cfg(windows)".dependencies.libc_errno]
-version = "0.3.10"
-default-features = false
-package = "errno"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.52, <0.62"
-features = [
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-]
-
-[target."cfg(windows)".dev-dependencies.once_cell]
-version = "1.20.3"
-
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
diff --git a/vendor/same-file-1.0.6/Cargo.toml b/vendor/same-file-1.0.6/Cargo.toml
index 4f66820..11ef472 100644
--- a/vendor/same-file-1.0.6/Cargo.toml
+++ b/vendor/same-file-1.0.6/Cargo.toml
@@ -25,5 +25,3 @@ license = "Unlicense/MIT"
 repository = "https://github.com/BurntSushi/same-file"
 [dev-dependencies.doc-comment]
 version = "0.3"
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.1"
diff --git a/vendor/snapbox-0.6.23/Cargo.toml b/vendor/snapbox-0.6.23/Cargo.toml
index 4c64f0f..cdbda12 100644
--- a/vendor/snapbox-0.6.23/Cargo.toml
+++ b/vendor/snapbox-0.6.23/Cargo.toml
@@ -84,7 +84,6 @@ cmd = [
     "dep:os_pipe",
     "dep:wait-timeout",
     "dep:libc",
-    "dep:windows-sys",
 ]
 color = [
     "dep:anstream",
@@ -222,11 +221,6 @@ version = "1.0.15"
 version = "0.2.176"
 optional = true
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.60.2"
-features = ["Win32_Foundation"]
-optional = true
-
 [lints.clippy]
 bool_assert_comparison = "allow"
 branches_sharing_code = "allow"
diff --git a/vendor/socket2-0.6.1/Cargo.toml b/vendor/socket2-0.6.1/Cargo.toml
index a1bacfd..db61e61 100644
--- a/vendor/socket2-0.6.1/Cargo.toml
+++ b/vendor/socket2-0.6.1/Cargo.toml
@@ -93,12 +93,3 @@ path = "src/lib.rs"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.172"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.60"
-features = [
-    "Win32_Foundation",
-    "Win32_Networking_WinSock",
-    "Win32_System_IO",
-    "Win32_System_Threading",
-    "Win32_System_WindowsProgramming",
-]
diff --git a/vendor/stacker-0.1.21/Cargo.toml b/vendor/stacker-0.1.21/Cargo.toml
index 14caa1a..08bfe2d 100644
--- a/vendor/stacker-0.1.21/Cargo.toml
+++ b/vendor/stacker-0.1.21/Cargo.toml
@@ -60,10 +60,3 @@ version = "0.1.7"
 [build-dependencies.cc]
 version = "1.1.22"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.52.0, <0.60.0"
-features = [
-    "Win32_System_Memory",
-    "Win32_System_Threading",
-    "Win32_Foundation",
-]
diff --git a/vendor/sysinfo-0.37.2/Cargo.toml b/vendor/sysinfo-0.37.2/Cargo.toml
index 2a586de..56404c7 100644
--- a/vendor/sysinfo-0.37.2/Cargo.toml
+++ b/vendor/sysinfo-0.37.2/Cargo.toml
@@ -48,24 +48,10 @@ cargo-args = ["-Zbuild-std"]
 rustdoc-args = ["--generate-link-to-definition"]
 
 [features]
-apple-app-store = ["apple-sandbox"]
+apple-app-store = []
 apple-sandbox = []
 c-interface = ["default"]
 component = [
-    "windows/Win32_Foundation",
-    "windows/Win32_Security",
-    "windows/Win32_System_Com",
-    "windows/Win32_System_Ole",
-    "windows/Win32_System_Rpc",
-    "windows/Win32_System_Variant",
-    "windows/Win32_System_Wmi",
-    "objc2-core-foundation/CFArray",
-    "objc2-core-foundation/CFBase",
-    "objc2-core-foundation/CFDictionary",
-    "objc2-core-foundation/CFNumber",
-    "objc2-core-foundation/CFString",
-    "objc2-io-kit",
-    "objc2-io-kit/hidsystem",
 ]
 debug = ["libc/extra_traits"]
 default = [
@@ -76,65 +62,17 @@ default = [
     "user",
 ]
 disk = [
-    "windows/Win32_Foundation",
-    "windows/Win32_Storage_FileSystem",
-    "windows/Win32_Security",
-    "windows/Win32_System_IO",
-    "windows/Win32_System_Ioctl",
-    "windows/Win32_System_SystemServices",
-    "windows/Win32_System_WindowsProgramming",
-    "objc2-core-foundation/CFArray",
-    "objc2-core-foundation/CFBase",
-    "objc2-core-foundation/CFDictionary",
-    "objc2-core-foundation/CFError",
-    "objc2-core-foundation/CFNumber",
-    "objc2-core-foundation/CFString",
-    "objc2-core-foundation/CFURL",
-    "objc2-io-kit",
 ]
 linux-netdevs = []
 linux-tmpfs = []
 multithread = ["dep:rayon"]
 network = [
-    "windows/Win32_Foundation",
-    "windows/Win32_NetworkManagement_IpHelper",
-    "windows/Win32_NetworkManagement_Ndis",
-    "windows/Win32_Networking_WinSock",
 ]
 system = [
-    "windows/Win32_Foundation",
-    "windows/Win32_System_Diagnostics_ToolHelp",
-    "windows/Wdk_System_SystemInformation",
-    "windows/Wdk_System_SystemServices",
-    "windows/Wdk_System_Threading",
-    "windows/Win32_Security_Authorization",
-    "windows/Win32_System_Diagnostics_Debug",
-    "windows/Win32_System_Kernel",
-    "windows/Win32_System_Memory",
-    "windows/Win32_System_Performance",
-    "windows/Win32_System_Power",
-    "windows/Win32_System_ProcessStatus",
-    "windows/Win32_System_Registry",
-    "windows/Win32_System_RemoteDesktop",
-    "windows/Win32_System_SystemInformation",
-    "windows/Win32_System_SystemServices",
-    "windows/Win32_System_Threading",
-    "windows/Win32_UI_Shell",
-    "dep:ntapi",
     "dep:memchr",
-    "objc2-core-foundation/CFBase",
-    "objc2-core-foundation/CFData",
-    "objc2-core-foundation/CFDictionary",
-    "objc2-core-foundation/CFString",
-    "objc2-io-kit",
 ]
 unknown-ci = []
 user = [
-    "windows/Win32_Foundation",
-    "windows/Win32_NetworkManagement_NetManagement",
-    "windows/Win32_Security",
-    "windows/Win32_Security_Authentication_Identity",
-    "windows/Win32_Security_Authorization",
 ]
 
 [lib]
@@ -212,28 +150,5 @@ version = "1.0"
 [dev-dependencies.tempfile]
 version = "3.9"
 
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.objc2-core-foundation]
-version = "0.3.1"
-features = ["std"]
-optional = true
-default-features = false
-
-[target.'cfg(any(target_os = "macos", target_os = "ios"))'.dependencies.objc2-io-kit]
-version = "0.3.1"
-features = [
-    "std",
-    "libc",
-]
-optional = true
-default-features = false
-
 [target.'cfg(not(any(target_os = "unknown", target_arch = "wasm32")))'.dependencies.libc]
 version = "^0.2.173"
-
-[target."cfg(windows)".dependencies.ntapi]
-version = "0.4"
-optional = true
-
-[target."cfg(windows)".dependencies.windows]
-version = ">=0.59, <0.62"
-optional = true
diff --git a/vendor/tempfile-3.23.0/Cargo.toml b/vendor/tempfile-3.23.0/Cargo.toml
index 0a4ceff..69ee1a5 100644
--- a/vendor/tempfile-3.23.0/Cargo.toml
+++ b/vendor/tempfile-3.23.0/Cargo.toml
@@ -88,10 +88,3 @@ features = ["fs"]
 version = "0.3.0"
 optional = true
 default-features = false
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = ">=0.52, <0.62"
-features = [
-    "Win32_Storage_FileSystem",
-    "Win32_Foundation",
-]
diff --git a/vendor/term-0.7.0/Cargo.toml b/vendor/term-0.7.0/Cargo.toml
index e89261e..ad5d62b 100644
--- a/vendor/term-0.7.0/Cargo.toml
+++ b/vendor/term-0.7.0/Cargo.toml
@@ -28,12 +28,6 @@ version = "2"
 
 [features]
 default = []
-[target."cfg(windows)".dependencies.rustversion]
-version = "1"
-
-[target."cfg(windows)".dependencies.winapi]
-version = "0.3"
-features = ["consoleapi", "wincon", "handleapi", "fileapi"]
 [badges.appveyor]
 repository = "Stebalien/term"
 
diff --git a/vendor/termcolor-1.4.1/Cargo.toml b/vendor/termcolor-1.4.1/Cargo.toml
index f90eaca..d32db67 100644
--- a/vendor/termcolor-1.4.1/Cargo.toml
+++ b/vendor/termcolor-1.4.1/Cargo.toml
@@ -35,6 +35,3 @@ name = "termcolor"
 bench = false
 
 [dev-dependencies]
-
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.3"
diff --git a/vendor/terminal_size-0.4.3/Cargo.toml b/vendor/terminal_size-0.4.3/Cargo.toml
index d6bef4e..ce13d3b 100644
--- a/vendor/terminal_size-0.4.3/Cargo.toml
+++ b/vendor/terminal_size-0.4.3/Cargo.toml
@@ -45,10 +45,3 @@ path = "examples/get_size.rs"
 [target."cfg(unix)".dependencies.rustix]
 version = "1.0.1"
 features = ["termios"]
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.60.0"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
diff --git a/vendor/termize-0.2.1/Cargo.toml b/vendor/termize-0.2.1/Cargo.toml
index 275d661..7938768 100644
--- a/vendor/termize-0.2.1/Cargo.toml
+++ b/vendor/termize-0.2.1/Cargo.toml
@@ -51,13 +51,6 @@ path = "examples/basic.rs"
 [target."cfg(unix)".dependencies.libc]
 version = "0.2.174"
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.0"
-features = [
-    "Win32_Foundation",
-    "Win32_System_Console",
-]
-
 [profile.dev]
 opt-level = 0
 lto = false
diff --git a/vendor/tokio-1.48.0/Cargo.toml b/vendor/tokio-1.48.0/Cargo.toml
index 05e6bcb..a4c8d12 100644
--- a/vendor/tokio-1.48.0/Cargo.toml
+++ b/vendor/tokio-1.48.0/Cargo.toml
@@ -99,11 +99,6 @@ net = [
     "mio/os-ext",
     "mio/net",
     "socket2",
-    "windows-sys/Win32_Foundation",
-    "windows-sys/Win32_Security",
-    "windows-sys/Win32_Storage_FileSystem",
-    "windows-sys/Win32_System_Pipes",
-    "windows-sys/Win32_System_SystemServices",
 ]
 process = [
     "bytes",
@@ -112,9 +107,6 @@ process = [
     "mio/os-ext",
     "mio/net",
     "signal-hook-registry",
-    "windows-sys/Win32_Foundation",
-    "windows-sys/Win32_System_Threading",
-    "windows-sys/Win32_System_WindowsProgramming",
 ]
 rt = []
 rt-multi-thread = ["rt"]
@@ -124,8 +116,6 @@ signal = [
     "mio/net",
     "mio/os-ext",
     "signal-hook-registry",
-    "windows-sys/Win32_Foundation",
-    "windows-sys/Win32_System_Console",
 ]
 sync = []
 taskdump = ["dep:backtrace"]
@@ -897,17 +887,6 @@ features = [
 ]
 default-features = false
 
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61"
-optional = true
-
-[target."cfg(windows)".dev-dependencies.windows-sys]
-version = "0.61"
-features = [
-    "Win32_Foundation",
-    "Win32_Security_Authorization",
-]
-
 [lints.rust.unexpected_cfgs]
 level = "warn"
 priority = 0
diff --git a/vendor/walkdir-2.5.0/Cargo.toml b/vendor/walkdir-2.5.0/Cargo.toml
index 4fda2f4..1741677 100644
--- a/vendor/walkdir-2.5.0/Cargo.toml
+++ b/vendor/walkdir-2.5.0/Cargo.toml
@@ -39,9 +39,6 @@ version = "1.0.1"
 [dev-dependencies.doc-comment]
 version = "0.3"
 
-[target."cfg(windows)".dependencies.winapi-util]
-version = "0.1.1"
-
 [badges.appveyor]
 repository = "BurntSushi/walkdir"
 
diff --git a/vendor/wasm-component-ld-0.5.19/Cargo.toml b/vendor/wasm-component-ld-0.5.19/Cargo.toml
index d1842b1..8091810 100644
--- a/vendor/wasm-component-ld-0.5.19/Cargo.toml
+++ b/vendor/wasm-component-ld-0.5.19/Cargo.toml
@@ -86,10 +86,3 @@ version = "0.241.0"
 
 [target."cfg(unix)".dependencies.libc]
 version = "0.2"
-
-[target."cfg(windows)".dependencies.windows-sys]
-version = "0.61.2"
-features = ["Win32_Foundation"]
-
-[target."cfg(windows)".dependencies.winsplit]
-version = "0.1"
diff --git a/vendor/yansi-term-0.1.2/Cargo.toml b/vendor/yansi-term-0.1.2/Cargo.toml
index 0317866..88ce8ef 100644
--- a/vendor/yansi-term-0.1.2/Cargo.toml
+++ b/vendor/yansi-term-0.1.2/Cargo.toml
@@ -36,9 +36,6 @@ version = "1.0"
 
 [features]
 derive_serde_style = ["serde"]
-[target."cfg(target_os=\"windows\")".dependencies.winapi]
-version = "0.3.4"
-features = ["consoleapi", "errhandlingapi", "fileapi", "handleapi", "processenv"]
 [badges.maintenance]
 status = "actively-developed"
 
